doc/2/api/controllers/document/update-by-query/index.md

```diff
@@ -0,0 +1,109 @@
+---
+code: true
+type: page
+title: updateByQuery
+---
+
+# updateByQuery
+
+Updates documents matching the provided search query. 
+
+Documents updated that way trigger real-time notifications.
```

Comment:
IMHO it's not necessary to specify that this method will trigger realtime notifications because all API method should trigger notifications

Reply:
It's a reminder, and I wrote it to be consistent with other method like
https://docs.kuzzle.io/core/2/api/controllers/document/delete-by-query/